### DISCUSSION TITLE
ComfyUI poor performance - new install

### DISCUSSION THREAD

**Discussion by EmbraceOfAI** [Comfy-Org/ComfyUI#6171] [General]
First time user here. I have installed ComfyUI within Pinokio and am experiencing very slow processing through the checkpoint / sampler and am getting very blurry or pixilated results.

XPS 8950
i7-12700K 
32GB RAM
Quadro P5000 (16GB)
2TB SSD Drive

The attached example took 287sec to run. Any suggestions on where to start troubleshooting?

![2](https://github.com/user-attachments/assets/71e541dd-05de-4085-9312-9115362a25f2)
![1](https://github.com/user-attachments/assets/ce7cfd92-4f52-4202-93a3-86a03bd4eb62)

**Comment by EmbraceOfAI** [comment]
I am also noticing that, after processing the image, my GPU memory is remaining nearly full.

Here is another example (after a reboot) that took 614 seconds to process.

![ComfyUI_00004_](https://github.com/user-attachments/assets/222caa82-90ed-422e-a4fd-77b238318f22)

**Comment by comfyanonymous** [comment]
This is a slow pascal card and you are trying flux one of the most demanding models.

If you really want to use Flux try these: https://comfyanonymous.github.io/ComfyUI_examples/flux/

Or else you are better off using a smaller model like SDXL or SD3.5 medium.

**Reply by EmbraceOfAI** [reply]
> This is a slow pascal card and you are trying flux one of the most demanding models.
> 
> If you really want to use Flux try these: https://comfyanonymous.github.io/ComfyUI_examples/flux/
> 
> Or else you are better off using a smaller model like SDXL or SD3.5 medium.

Got it, that makes sense thank you. I think I’ll set up a box with a more appropriate GPU and keep it separate from my workstation.
